.github/workflows/release.yml

```
@@ -0,0 +1,77 @@
+        with:
+          go-version-file: go.mod
+          cache: true
+          
+      - name: Build Linux binary
+        if: steps.check.outputs.skip == 'false'
+        run: |
+          GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o buzz-linux-amd64 .
+          
+      - name: Create Release
+        if: steps.check.outputs.skip == 'false'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)